lxc/auth.go

```diff
@@ -1308,7 +1303,17 @@ func (c *cmdPermissionList) run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	permissionsInfo, err := resource.server.GetPermissionsInfo(lxd.GetPermissionsArgs{
+	// If no remote was specified, use the default remote.
+	if client == nil {
```

Comment:
Can we move this logic up above the project logic so its together with the other client logic?

Reply:
Done. I've used similar logic to the example you posted.